File: Backup/PlaceYourBets/PredictionLeagueTableForm.vb
Public Class PredictionLeagueTableForm

    Private userResultList As New List(Of Users)

    Private Sub MainMenuButton_Click(sender As System.Object, e As System.EventArgs) Handles MainMenuButton.Click
        MenuForm.Show()
        Me.Close()
    End Sub

    Private Sub ShutdownButton_Click(sender As System.Object, e As System.EventArgs)
        Application.Exit()
    End Sub

    Private Sub PredictionLeagueTableForm_Load(sender As System.Object, e As System.EventArgs) Handles MyBase.Load
        Dim fb As New FixtureBuilder
        userResultList = fb.getScores()

        Me.PredictionsLeagueFlowLayoutPanel.Enabled = True
        Me.PredictionsLeagueFlowLayoutPanel.Visible = True
        Me.NavigationGroupBox.Visible = True
        Me.MainMenuButton.Visible = True

        For Each user As Users In userResultList
            Dim review As New PredictionLeagueUserControl(user.user, user.correct)
            Me.PredictionsLeagueFlowLayoutPanel.Controls.Add(review)

        Next

    End Sub
End Class

File: PlaceYourBets.ConvertedToC#/PredictionLeagueTableForm.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;
using System.Linq;
using System.Xml.Linq;
namespace PlaceYourBets
{
	public partial class PredictionLeagueTableForm
	{


		private List<Users> userResultList = new List<Users>();
		private void MainMenuButton_Click(System.Object sender, System.EventArgs e)
		{
			My.MyProject.Forms.MenuForm.Show();
			this.Close();
		}

		private void ShutdownButton_Click(System.Object sender, System.EventArgs e)
		{
			Application.Exit();
		}

		private void PredictionLeagueTableForm_Load(System.Object sender, System.EventArgs e)
		{
			FixtureBuilder fb = new FixtureBuilder();
			userResultList = fb.getScores();

			this.PredictionsLeagueFlowLayoutPanel.Enabled = true;
			this.PredictionsLeagueFlowLayoutPanel.Visible = true;
			this.NavigationGroupBox.Visible = true;
			this.MainMenuButton.Visible = true;

			foreach (Users user in userResultList) {
				PredictionLeagueUserControl review = new PredictionLeagueUserControl(user.user, user.correct);
				this.PredictionsLeagueFlowLayoutPanel.Controls.Add(review);

			}

		}
	}
}

